Sources/SwiftFormat/API/Configuration.swift

```diff
@@ -173,6 +174,22 @@ public struct Configuration: Codable, Equatable {
   /// Contains exceptions for the `NoAssignmentInExpressions` rule.
   public var noAssignmentInExpressions: NoAssignmentInExpressionsConfiguration
 
+  /// Determines how trailing commas in comma-separated lists should be handled during formatting.
+  public enum TrailingCommasInMultilineLists: String, Codable {
+    case always
+    case never
+    case ignore
+  }
+
+  /// Determines how trailing commas in comma-separated lists are handled during formatting.
+  ///
+  /// This setting takes precedence over `multiElementCollectionTrailingCommas`.
+  /// If set to `.ignore` (the default), the formatter defers to `multiElementCollectionTrailingCommas`
+  /// for collections only. In all other cases, existing trailing commas are preserved as-is and not modified.
+  /// If set to `.always` or `.never`, that behavior is applied uniformly across all list types,
+  /// regardless of `multiElementCollectionTrailingCommas`.
+  public var trailingCommasInMultilineLists: TrailingCommasInMultilineLists
```

Comment:
Looking at this more closely, this feels too similar to the name of the other property below. I don't know if `multilineLists` really communicates the bit about parameter lists, tuples, closure captures, etc.

What about `multilineTrailingCommaBehavior`? Then, let's rename the enum to `MultilineTrailingCommaBehavior` and have the cases be `alwaysUsed`, `neverUsed`, and `keptAsWritten`. I think that helps to better document what's going on.

Later on (not in this PR), we could add a fourth case: `onlyCollectionLiterals`, which could take over the behavior of the other Boolean setting, and deprecate that one.

Reply:
I understand. I’ll make sure to be careful when adding or changing rules. Thanks for explaining! 🙇